docs/topics/mta-web-assessment-module.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+// * topics/mta-assessment-module.adoc
+
+:_content-type: REFERENCE
+[id="mta-web-assessment-module_{context}"]
+= Assessment Module
+
+The assessment process for applications that are to be migrated in Red Hat Migration Toolkit for Applications (MTA) has been separated from other tool features and enhanced in version 7.0.0.
+
+* A separate new Assessment hub includes the Assessment module, replacing the older _Pathfinder_ feature.
+* Assessment supports multiple questionnaires, relevant to each assessed application.
+* Default questionnaires can be customized and saved.
+* User can design and version custom questionnaires and questions using YAML syntax.
+* Assessed applications are grouped into *archetypes* with similar application characteristics.
+* Assessment module integrates tightly with the application inventory.
+* Assessment questionnaires can be imported and exported.
+* Enhanced questionnaire capabilities:
+** Conditional questions
+** Application auto-tagging according to answers
+** Automated answers from tags in application archetypes
+
+In the MTA user interface, the Assessment Module appears in a separate menu item. Selecting the item shows the currently saved default and custom questionnaires.
+
+:_content-type: PROCEDURE
+[id="mta-assessment-start-questionnaire_{context}"]
+== Opening an assessment questionnaire
+
+The {ProductName} ({ProductShortName}) assesses applications that are to be migrated according to a set of questions relevant to the application, such as dependencies, *[more question types]*. {ProductShortName} comes with a default questionnaire/s that the user can edit, customize and save as new questionnaires.
+
+.To import an application
+
```

Comment:
Procedure titles are written as gerunds, procedures have the following structure, one per module:
= Doing x. [Importing an application]
.Prerequisites [if there are any]

.Procedure

. Step 1.
. Step 2.